topics/07_web_ui_testing/ui_test_text_box.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from selenium import webdriver
+from ui_text_box_page import TextBoxPage
+
+
+@pytest.fixture
+def driver():
+    """Fixture to set up"""
+    driver = webdriver.Chrome()
+    driver.maximize_window()
+    yield driver
+    driver.quit()
+
+
+def test_text_box_form_submission(driver):
+    """Tests validation"""
+    page = TextBoxPage(driver)
+    page.open()
+
+    """Test data"""
+    full_name = "Donald Duck"
+    email = "donald.duck@example.com"
+    current_address = "56 Main St"
+    permanent_address = "379 Apple Rd"
+
+    """Fill form"""
+    page.fill_form(full_name, email, current_address, permanent_address)
+    page.submit()
+
+    """Get output"""
```

Comment:
Triple quotes are typically used for docstrings and multiline comments, whereas `#` is customary for single-line comments.